index.html

```diff
@@ -0,0 +1,194 @@
+                <li><img src="img/Counter.png" alt="" class="dot"></li>
+                <li><a href="#" class="project-link">2015</a></li>
+
+            </ul>
+            <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+             <ul class="project-languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+                
+             </ul>
+             <button class="btn"><div class="senta">See Project<div></button>
+             
+        </div>
+        </section>
+
+        <section class="about-me">
+        <div class="about-me-container">
+            <section class="about-1">
+                 <h1 class="about-header">About Myself</h1>
+                 <p class="about-description"> Hello I'm a software devoloper! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, dont hesitate to contact me.</p>
+                 <p class="connect-2">LET'S CONNECT</p>
+                 <ul class="connect-icons">
+                    <li><a href="https://twitter.com/JosiahMutinhiri"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="https://www.linkedin.com/in/josiah-mutinhiri-536387153/"><i class="fa-brands fa-linkedin"></i></a></li>
+                    <li><a href="https://github.com/josman227"><i class="fa-sharp fa-solid fa-m"></i></i></a></li>
+                    <li><a href="https://www.facebook.com/josiah.mutinhiri"><i class="fa-brands fa-facebook"></i></a></li>
+                    <li><a href="https://github.com/josman227"><i class="fa-brands fa-github"></i></a></li>
+                    <li><a href="https://github.com/josman227"><i class="fa-brands fa-angellist"></i></a></li>
+                 </ul>
+                 <button class=" btn resume-btn">Get my resume</button>
+            </section>
+            <section class="about-2">
+             <ul class="tools">
+                     <div class="lang-arrow">
+                         <h3>Languages</h3>
+                         <img src="img/arrow.png" class="ira" alt="">
+                     </div>
+                     
+                     <ul class=langua>
+                         <li class="lang-item">
+                             <img class="ipin" src="img/javascript-logo.png" alt="lang">
+                             <p>JavaScript</p>
+                         </li>
+                         <li class="lang-item">
+                             <img class="ipin" src="img/html-logo.png" alt="lang">
+                             <p>HTML</p>
+                         </li>
+                         <li class="lang-item">
+                             <img class="ipin" src="img/css-logo.png" alt="lang"> 
+                             <p>CSS</p> 
+                         </li>
```

Comment:
I appreciate the effort you put into implementing the required changes. Good job 👍, 

However, there are still some issues that haven't been addressed yet. but don't worry, you are almost there. 🤗🤗🏃‍♂️

Your design
![image](https://user-images.githubusercontent.com/78251927/229294971-094fbd32-cca0-4cfb-ab0d-deb5989ab07b.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/229295000-01a751a0-860a-40c3-a15f-0caa6171aa5e.png)

As requested in the previous review 👉 https://github.com/josman227/Portfolio-Project/pull/1 kindly adjust the position of the icons `icons` and the names (JavaScript, HTML, and CSS) to match the Figma design. 👍